Xamarin.Forms.Platform.iOS/Renderers/ButtonRenderer.cs

```diff
@@ -88,6 +90,19 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateImage();
 		}
 
+		void SetControlPropertiesFromProxy()
+		{
+			var controlStates = new[] {UIControlState.Normal, UIControlState.Highlighted, UIControlState.Disabled};
```

Comment:
Can this array be made static on the ButtonRenderer class, rather than creating a new instance every time?